lib/dRep/util/kbase_obj.py

```diff
@@ -0,0 +1,249 @@
+import os
```

Comment:
Why's this file called kbase_obj?

Reply:
what are you supposed to call it? it wraps the stuff from `ws.get_objects2`